app/controllers/admin/volunteers.py

```diff
@@ -87,12 +89,10 @@ def addVolunteer(volunteer, volunteerEventID, eventLengthInHours):
         flash("Volunteer successfully added!", "success")
     else:
         flash("Error when adding volunteer", "danger")
-    return "" #must return something for ajax
-
-
+    return jsonify({"Success": True}), 200
 
 @admin_bp.route('/removeVolunteerFromEvent/<user>/<eventID>', methods = ['POST'])
 def removeVolunteerFromEvent(user, eventID):
     (EventParticipant.delete().where(EventParticipant.user == user, EventParticipant.event == eventID)).execute()
-    flash("Volunteer successfully removed")
-    return "Volunteer successfully removed"
+    flash("Volunteer successfully removed", "success")
+    return jsonify({"Success": True}), 200
```

Comment:
let's not do the `"Success" : True` thing if we're not actually checking the returned data for success or failure